examples/components_vertical/CCFRAUD/preprocessing/run.py

```diff
@@ -0,0 +1,202 @@
+import os
```

Comment:
let's put all components under components/

but you could rename your directory CCFRAUD_VERTICAL like you did for the pipelines